code/__DEFINES/subsystems.dm

```diff
@@ -114,6 +114,7 @@
 #define INIT_ORDER_QUIRKS			60
 #define INIT_ORDER_TICKER			55
 #define INIT_ORDER_MAPPING			50
+#define INIT_ORDER_BS_EXPLORATION	49 //MUST BE BEFORE MAPPING
```

Comment:
> must be before
> defines after

Reply:
Bigger is after isnt it?

Reply:
I don't _think_ so?

Reply:
Oh yea the comment is wrong and should say after